packages/terra-consumer-nav/src/Nav.scss

```diff
@@ -3,10 +3,12 @@
 
 :local {
   .nav-container {
+    background: inherit;
```

Comment:
As mention above making the profile background inherit from gradient causes the profile background to look different from the Nav background. It works as expected if the background is not gradient. Making the profile background transparent make the profile to have continuous gradient background but it causes an issue for scrolling. For now made profile background transparent for desktop and inherit from nav for mobile. If anyone has suggestion I appreciate.